src/include/mpitimpl.h

```diff
@@ -1192,6 +1192,8 @@ extern MPID_Thread_mutex_t mpi_t_mutex;
     do { \
         int err_; \
         MPIR_T_THREAD_CHECK_BEGIN \
+        MPID_Thread_init(&err_); \
```

Comment:
Can argobot init be called multiple times?

Reply:
Yes. It is okay to call it multiple times (internally there's a refcount).
If it wouldn't be the case, `MPL_thread_init()` will take care of it.

Reply:
This concern is mentioned in `src/mpl/include/mpl_thread_argobots.h`.